src/image/VisitInfo.cc

```diff
@@ -283,11 +303,23 @@ class VisitInfoFactory : public table::io::PersistableFactory {
                                                  record.get(keys.elevation)),
                               coord::Weather(record.get(keys.airTemperature), record.get(keys.airPressure),
                                              record.get(keys.humidity)),
-                              record.get(keys.instrumentLabel)));
+                              instrumentLabel));
         return result;
     }
 
     explicit VisitInfoFactory(std::string const& name) : table::io::PersistableFactory(name) {}
+
+private:
+    int getVersion(table::BaseRecord const& record) const {
```

Comment:
I suggest deferring that to another ticket, so that we can merge the fix before the weekly.

Note that it cannot be for free, because at the minimum you need to add a version key to every schema. I'm not even sure if `Persistable` requires that the format be a FITS table, or just any FITS.